src/key_value_store/kv_flat_btree_async.cc

```diff
@@ -15,11 +15,14 @@
 #include "key_value_store/kv_flat_btree_async.h"
 #include "key_value_store/kvs_arg_types.h"
 #include "include/rados/librados.hpp"
+#if defined(__linux__)
```

Comment:
```diff
diff --git a/src/key_value_store/kv_flat_btree_async.cc b/src/key_value_store/kv_flat_btree_async.cc
index cdf3ef28a9..b833a458d2 100644
--- a/src/key_value_store/kv_flat_btree_async.cc
+++ b/src/key_value_store/kv_flat_btree_async.cc
@@ -15,13 +15,11 @@
 #include "key_value_store/kv_flat_btree_async.h"
 #include "key_value_store/kvs_arg_types.h"
 #include "include/rados/librados.hpp"
-#include "/usr/include/asm-generic/errno.h"
-#include "/usr/include/asm-generic/errno-base.h"
 #include "common/ceph_context.h"
 #include "common/Clock.h"
 #include "include/types.h"

-
+#include <errno.h>
 #include <string>
 #include <iostream>
 #include <cassert>
```

instead of guarding these `#include`s , probably we should just `#include <errno.h>`. see #25256 .

Reply:
@tchaikov 
ATM I do not have a Linux system where I can build ceph, so I have to trust you on this that it'll work. It will work for FreeBSD. 
If you merge #25256 first, then I'll repair this PR